utils/api.js

```diff
@@ -37,4 +37,14 @@ export const patchArticleVote = (id, likeAmount)=>{
     })
 }
 
-
+export const postsNewComment = (id, newComment)=>{
+    const commentToAdd = {
+        username: 'happyamy2016',
+        body: newComment
+    }
+    return app.post(`/articles/${id}/comments`, commentToAdd).then(({data})=>{
+        return data.comment
+    }).catch((err)=>{
+        next(err)
```

Comment:
No use for next here as we don't have any error handling middleware so remove this here and above please